subsys/usb/usb_c/usbc_prl.c

```diff
@@ -1175,9 +1175,11 @@ static void prl_rx_wait_for_phy_message(const struct device *dev)
 	int8_t msid;
 	uint8_t num_data_objs;
 	uint8_t power_role;
+	int pending_rx;
 
 	/* Get the message */
-	if (tcpc_receive_data(tcpc, rx_emsg) <= 0) {
+	pending_rx = tcpc_is_rx_pending_msg(tcpc, NULL);
+	if (pending_rx == -ENODATA || tcpc_receive_data(tcpc, rx_emsg) <= 0) {
```

Comment:
Even though you're deleting `tcpc_is_rx_pending_msg()` in another commit, the logic here isn't correct and would break bisectability.

```suggestion
	if (pending_rx == 0 || tcpc_receive_data(tcpc, rx_emsg) <= 0) {
```

Reply:
Right, missed this one, thanks!